model/src/main/kotlin/config/Excludes.kt

```diff
@@ -66,30 +77,35 @@ data class Excludes(
      * excluded. If the [id] references a [Project] it checks if the [Project] is excluded and if the [id] does not
      * appear as a dependency of another non-excluded [Project].
      */
-    fun isExcluded(id: Identifier, analyzerResult: AnalyzerResult) =
-            analyzerResult.projects.find { it.id == id }?.let {
+    fun isExcluded(id: Identifier, ortResult: OrtResult) =
+            ortResult.analyzer?.result?.projects?.find { it.id == id }?.let {
                 // An excluded project could still be included as a dependency of another non-excluded project.
-                isProjectExcluded(it) && isPackageExcluded(id, analyzerResult)
-            } ?: isPackageExcluded(id, analyzerResult)
+                isProjectExcluded(it, ortResult) && isPackageExcluded(id, ortResult)
+            } ?: isPackageExcluded(id, ortResult)
 
     /**
-     * True if all occurrences of the package identified by [id] in the [analyzerResult] are excluded by this [Excludes]
+     * True if all occurrences of the package identified by [id] in the [ortResult] are excluded by this [Excludes]
      * configuration.
      */
-    fun isPackageExcluded(id: Identifier, analyzerResult: AnalyzerResult) =
-            analyzerResult.projects.all { project ->
-                isProjectExcluded(project) || project.scopes.all { scope ->
+    fun isPackageExcluded(id: Identifier, ortResult: OrtResult) =
+            ortResult.analyzer?.result?.projects?.all { project ->
+                isProjectExcluded(project, ortResult) || project.scopes.all { scope ->
                     isScopeExcluded(scope, project) || !scope.contains(id)
                 }
-            }
+            } ?: true
+
+    /**
+     * True if any [path exclude][paths] matches [path].
+     */
+    fun isPathExcluded(path: String) = paths.any { it.matches(path) }
 
     /**
      * True if the [project] is excluded by this [Excludes] configuration.
      */
-    fun isProjectExcluded(project: Project) =
+    fun isProjectExcluded(project: Project, ortResult: OrtResult) =
             projects.any {
                 it.matches(project.definitionFilePath) && it.isWholeProjectExcluded
-            }
+            } || paths.any { it.matches(ortResult.getDefinitionFilePathRelativeToAnalyzerRoot(project)) }
```

Comment:
Nit: Maybe reformat this now like

            projects.any { it.matches(project.definitionFilePath) && it.isWholeProjectExcluded } ||
                    paths.any { it.matches(ortResult.getDefinitionFilePathRelativeToAnalyzerRoot(project)) }

Reply:
Done.